### PR TITLE
chore: add french translations to the changelog

### PR DESCRIPTION
**Pull request by daine**
# Summary | Résumé

This PR adds [the french translations](https://docs.google.com/document/d/1DLsbKgTMzxyDpVoqnLlbsuzVO0pnN0Nw5Bws6QKjRRg/edit?tab=t.0) for the release notes on the changelog file

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -184,6 +184,90 @@ Released on: 2023-02-10
...
+  - `margin` (marge)
+  - `padding` (remplissage)
+ 
+Nous avons mis à jour les valeurs des classes d'espacement. Bien que certaines classes restent encore, leur valeur sous-jacente en rem ou en pixels a changé. Vérifiez la section d'espacement de notre nouvelle publication d'unités de style pour les valeurs de pixel mises à jour.
+
+
+_Exemple_: Si vous utilisiez auparavant mt-400 pour une marge supérieure de 400, elle sera mise à jour à 32px. Si vous souhaitez conserver sa taille précédente de 24px, vous devez modifier votre code pour utiliser mt-300 à la place. Voici un [guide de cartographie visuelle](https://github.com/cds-snc/gcds-tokens/blob/main/CHANGELOG.md#spacing-tokens-visual-mapping-guide) pour vous aider à identifier les valeurs que vous devez utiliser.
+    
+### Changement de typographie
+* Toutes les classes de légende ont été dépréciées et remplacées par `text-small`, y compris leurs variations pour tous les différents affichages et états :
```
Comment (daine) [inline CHANGELOG.md:214]:
```suggestion
* Toutes les classes `caption` ont été dépréciées et remplacées par `text-small`, y compris leurs variations pour tous les différents affichages et états :
```
@SmartMouthWords does this make sense?

In english we're saying all `caption` classes -- caption is code, should I keep it here too? Here is the english version:
https://github.com/cds-snc/gcds-utility/blob/main/CHANGELOG.md#typography-changes

File: CHANGELOG.md [hunk 2]
```diff
@@ -184,6 +184,90 @@ Released on: 2023-02-10
...
+* mettre à jour le cadre utilitaire avec de nouvelles typographies + unités de style d'espacement ([#197](https://github.com/cds-snc/gcds-utility/issues/197)) ([f6c2a24](https://github.com/cds-snc/gcds-utility/commit/f6c2a248547be78cb800fbb5770de99e457d8e30))
+
+Dans le cadre de nos efforts continus pour s'aligner sur les spécifications de Canada.ca, nous avons mis à jour nos échelles d'espacement et de typographie. Ces mises à jour ont été introduites avec [la dernière version de nos unités de style de conception](https://github.com/cds-snc/gcds-tokens/blob/main/CHANGELOG.md#200).
+
+### Changement d’espacement
+
+* De nouvelles classes d'espacement sont maintenant disponibles
+* Les valeurs de toutes les classes précédentes ont changé
+  - `margin` (marge)
+  - `padding` (remplissage)
```
Comment (daine) [inline CHANGELOG.md:206]:
```suggestion
  - `margin` (marge)
  - `padding` (remplissage)
```

I added the french in parenthesis, originally it was:
```suggestion
  - marge
  - remplissage
```

File: CHANGELOG.md (hunk 1 above)
Comment (SmartMouthWords) [reply CHANGELOG.md:214]: Just checked and we've been using the term "caption' in both French and English, so "de légende" can be replaced by "de caption"

File: CHANGELOG.md (hunk 2 above)
Comment (SmartMouthWords) [reply CHANGELOG.md:206]: Yes but can you swap out "remplissage" for "marges intérieures"? The translation team uses "marge intérieure" for "padding".

Comment (daine) [reply CHANGELOG.md:206]:
```suggestion
  - `margin` (marge)
  - `padding` (marges intérieures)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -184,6 +184,90 @@ Released on: 2023-02-10
 
 Tout changement important à ce projet sera consigné dans le présent fichier.
 
+## [1.5.0](https://github.com/cds-snc/gcds-utility/compare/gcds-utility-v1.4.0...gcds-utility-v1.5.0)
+
+Publié le :  2024-12-05
+
+### :rocket: Nouvelles fonctionnalités
+
+* add additional display classes ([#191](https://github.com/cds-snc/gcds-utility/issues/191)) ([7917aa1](https://github.com/cds-snc/gcds-utility/commit/7917aa1515cb05ef05dc64044775aa7e983f5c5f))
```
Comment (daine) [inline CHANGELOG.md:193]:
```suggestion
* Ajouter de nouvelles classes d'affichage ([#191](https://github.com/cds-snc/gcds-utility/issues/191)) ([7917aa1](https://github.com/cds-snc/gcds-utility/commit/7917aa1515cb05ef05dc64044775aa7e983f5c5f))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -184,6 +184,90 @@ Released on: 2023-02-10
 
 Tout changement important à ce projet sera consigné dans le présent fichier.
 
+## [1.5.0](https://github.com/cds-snc/gcds-utility/compare/gcds-utility-v1.4.0...gcds-utility-v1.5.0)
+
+Publié le :  2024-12-05
```
Comment (SmartMouthWords) [inline CHANGELOG.md:189]:
```suggestion
Publié le :  2024-12-05
```
Version publiée le :  2024-12-05

File: CHANGELOG.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ All notable changes to this project will be documented in this file.
 
 ### :rocket: New Features
 
-* add additional display classes ([#191](https://github.com/cds-snc/gcds-utility/issues/191)) ([7917aa1](https://github.com/cds-snc/gcds-utility/commit/7917aa1515cb05ef05dc64044775aa7e983f5c5f))
+* Ajouter de nouvelles classes d'affichage ([#191](https://github.com/cds-snc/gcds-utility/issues/191) en anglais) ([7917aa1](https://github.com/cds-snc/gcds-utility/commit/7917aa1515cb05ef05dc64044775aa7e983f5c5f))
```
Comment (daine) [inline CHANGELOG.md:12]:
```suggestion
* add additional display classes ([#191](https://github.com/cds-snc/gcds-utility/issues/191) en anglais) ([7917aa1](https://github.com/cds-snc/gcds-utility/commit/7917aa1515cb05ef05dc64044775aa7e983f5c5f))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -9,7 +9,7 @@ All notable changes to this project will be documented in this file.
 
 ### :rocket: New Features
 
-* add additional display classes ([#191](https://github.com/cds-snc/gcds-utility/issues/191)) ([7917aa1](https://github.com/cds-snc/gcds-utility/commit/7917aa1515cb05ef05dc64044775aa7e983f5c5f))
+* add additional display classes ([#191](https://github.com/cds-snc/gcds-utility/issues/191) en anglais) ([7917aa1](https://github.com/cds-snc/gcds-utility/commit/7917aa1515cb05ef05dc64044775aa7e983f5c5f))
```
Comment (daine) [inline CHANGELOG.md:12]:
```suggestion
* add additional display classes ([#191](https://github.com/cds-snc/gcds-utility/issues/191)) ([7917aa1](https://github.com/cds-snc/gcds-utility/commit/7917aa1515cb05ef05dc64044775aa7e983f5c5f))
```

**Review by SmartMouthWords — DISMISSED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (daine) [inline CHANGELOG.md:189]:
```suggestion
Version publiée le :  2024-12-05
```

**Review by SmartMouthWords — APPROVED** [review]

**Pull request merged**